.github/workflows/tests.yaml

```diff
@@ -88,6 +88,7 @@ jobs:
               run: |
                   ./scripts/run_in_build_env.sh \
                   "./scripts/tests/run_test_suite.py \
+                     --target-skip-glob '{TestGroupDemoCommand,TestGroupDemoConfig}' \
```

Comment:
Please add some comment explaining why these demo tests are globbed out of CI everywhere.

If CI does not execute any of these tests, the tests will easily get broken in time

Reply:
In each of the test files, there is an explanation on why it shouldn't be run by the CI. Both test files are only used for manual group tests to simplify testing for manifacturers. The actual group messaging tests are done by `TestGroupMessaging`.